compiler/src/iree/compiler/Codegen/Common/TileAndDistributeToWorkgroupsPass.cpp

```diff
@@ -104,6 +104,14 @@ static LogicalResult getTileAndDistributeConfig(
     tileSizes[loopId] = 0;
   }
 
+  // Loops with a tile size of 0 shouldn't be distributed.
```

Comment:
AFAIR this should already be happening. Can you provide example of what is hitting this and add a test maybe?

Reply:
The test I added hits this case

Reply:
If the tile size is zero, it shouldnt even be tiled... so maybe the issue is somewhere else?

Reply:
add a test to show the problem. Please take another look

Reply:
IIUC, if you add
```
if (tileSizes[partitionedLoop] == 0) continue;
```
in the loop  here
https://github.com/openxla/iree/blob/289b9a1ebf954be3995cc1895254d09f2623341f/compiler/src/iree/compiler/Codegen/Common/TileAndDistributeToWorkgroupsPass.cpp#L205 it should also fix?

Reply:
yes that works, changed it.